runelite-client/src/main/resources/net/runelite/client/plugins/gpu/vert.glsl

```diff
@@ -23,20 +23,61 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#version 330
+#version 400
```

Comment:
any reason why this was changed?

Reply:
at any rate it's no longer necessary, so I reverted it